games/Powermonger/powermonger.asm

```diff
@@ -31,30 +31,44 @@
 	ENDC
 
 ;DEBUG
+;QUICKSTART=1
+;SHOWTIME=1
+EXPSIZE=$80000
 
 ;======================================================================
 
 _base		SLAVE_HEADER				;ws_Security + ws_ID
-		dc.w	10				;ws_Version
+		dc.w	18				;ws_Version
 		dc.w	WHDLF_Disk|WHDLF_NoError|WHDLF_NoDivZero	;ws_flags
 		dc.l	$80000				;ws_BaseMemSize
-		dc.l	$4e800				;ws_ExecInstall
+		dc.l	0				;ws_ExecInstall
 		dc.w	_Start-_base			;ws_GameLoader
 		dc.w	0				;ws_CurrentDir
 		dc.w	0				;ws_DontCache
 _keydebug	dc.b	0				;ws_keydebug = F9
 _keyexit	dc.b	$59				;ws_keyexit = F10
-_expmem		dc.l	0				;ws_ExpMem
+_expmem		dc.l	-EXPSIZE			;ws_ExpMem
 		dc.w	_name-_base			;ws_name
 		dc.w	_copy-_base			;ws_copy
 		dc.w	_info-_base			;ws_info
+		dc.w	0				;kickstart name
+		dc.l	0				;kicksize
+		dc.w	0				;kickcrc
+		dc.w	_config-_base
+_config
+		dc.b	"C1:L:Fast RAM use:Auto,Off,On;"
+		; C2 is frame rate limit
+		dc.b	"C3:X:All levels at start:0;"
```

Comment:
what means 'All levels at start'?

Reply:
If you start the game with this option enabled all maps (maybe levels is the wrong word) are available. I.e. if you enable both trainer options you can start a new game, scroll to the bottom to find the last map and win it immediately when you retire (I used this to test the end sequence).

I'm open to suggestions for calling it something else.